CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# QA approval for 1% and stable promotions
+configs/versions.json    @JuliaTrubchyk
```

Comment:
@rsimha thoughts on creating a QA team instead?

Reply:
Good idea. I just created https://github.com/orgs/ampproject/teams/amp-qa and invited @JuliaTrubchyk to the team. Let me know if this works.

Reply:
It should actually just be QA. Thanks!

Reply:
Gotcha. Renamed to https://github.com/orgs/ampproject/teams/qa.

Reply:
Wait haha I meant the members of the team, not the name. amp-qa is fine!

Reply:
🤦 Renamed back to `amp-qa`.